### PR TITLE
Docs: spell checker update.

### PR DESCRIPTION
**Pull request by godai78**
Docs: Updating the spell checker documentation.

Ref: [cksource/content-documentation-proofread#358](https://github.com/cksource/content-documentation-proofread/issues/358)

### CODE REVIEW

File: docs/features/spellcheck/README.md [hunk 1]
```diff
@@ -56,17 +56,18 @@ The [WebSpellChecker Dialog](https://ckeditor.com/cke4/addon/wsc) plugin is anot
     This is a commercial solution provided by our partner, [WebSpellChecker](https://webspellchecker.com/). You can report any issues in its [GitHub repository](https://github.com/WebSpellChecker/wproofreader).
 </info-box>
 
-[WProofreader](https://webspellchecker.com/wsc-proofreader) is an innovative proofreading tool that combines the functionality of "spell check as you type" and "spell check in a dialog" in a modern UI. Spelling and grammar suggestions are available on hover with no clicking needed.
+[WProofreader](https://webspellchecker.com/wsc-proofreader) is an innovative proofreading tool that combines the functionality of "spell check as you type" and "spell check in a dialog" in a modern UI. Spelling, punctuation and grammar suggestions are available on hover with no clicking needed.
```
Comment (AnnaTomanek) [inline docs/features/spellcheck/README.md:59]:
```suggestion
[WProofreader](https://webspellchecker.com/wsc-proofreader) is an innovative proofreading tool that combines the functionality of "spell check as you type" and "spell check in a dialog" in a modern UI. Spelling, punctuation and grammar suggestions are available on hover with no clicking needed or as a convenient dialog, both with additional in-place replacement suggestions.
```

File: docs/features/spellcheck/README.md [hunk 2]
```diff
@@ -56,17 +56,18 @@ The [WebSpellChecker Dialog](https://ckeditor.com/cke4/addon/wsc) plugin is anot
...
 
 {@img assets/img/wproofreader_01.png 730 Spelling and grammar checking with WProofreader in CKEditor 4 WYSIWYG editor}
 
-The distraction-free badge gives you access to proofreader suggestions, settings and proofreading overview in a dialog.
+The proofreader badge in the bottom-right corner shows you the total number of mistakes detected. Hover an underlined word to display the proofreader suggestions for any of the spelling and grammar mistakes found. This hover card allows the user to employ the feature on the go.
```
Comment (AnnaTomanek) [inline docs/features/spellcheck/README.md:63]:
```suggestion
The proofreader badge in the bottom-right corner shows you the total number of mistakes detected. Hover an underlined word to display the proofreader suggestions for any of the spelling and grammar mistakes found. This hovercard allows the user to employ the feature on the go.
```

File: docs/features/spellcheck/README.md [hunk 3]
```diff
@@ -92,9 +93,29 @@ For more detailed documentation, refer to the [official WProofreader "Getting St
...
+
+There are also two types of custom dictionaries available.
+
+One is the **user dictionary** that can be expanded and will grow during the regular use of the feature simply by adding new words to the dictionary. This is a perfect solution for editors and writers working on specific content that may contain slang or professional jargon.
+
+The other is the so-called **company-wide dictionary**. These premade dictionaries can be uploaded by the system administrators or CKEditor 5 integrators and can be made available company-wide, accessible for all users. This way all benefits of a user-generated dictionary can be shared among the team, making the proofreading process more structured and controlled.
+
+### Multi-language Support
+
+For content in multiple languages, it is also possible to use WProofreader. The spell check feature can automatically detect the text language properly. It is enough to choose “Auto Detect” in the Language dropdown in the WProofreader settings. Suggestions for spelling and grammar (where available) will be displayed properly for each text fragment.
```
Comment (AnnaTomanek) [inline docs/features/spellcheck/README.md:116]: As a matter of fact, this is supported in both WProofreader and SCAYT.

Comment (godai78) [reply docs/features/spellcheck/README.md:116]: Let's remove the first sentence, then.

Comment (godai78) [inline docs/features/spellcheck/README.md:116]:
```suggestion
If yoy have content in multiple languages, the spell check feature can automatically detect the text language properly. It is enough to choose “Auto Detect” in the Language dropdown in the WProofreader settings. Suggestions for spelling and grammar (where available) will be displayed properly for each text fragment.
```

File: docs/features/spellcheck/README.md [hunk 4]
```diff
@@ -92,9 +93,29 @@ For more detailed documentation, refer to the [official WProofreader "Getting St
 
 ## Supported Languages
 
-By default the WebSpellChecker solutions support 17 languages for spell checking: American English, British English, Brazilian Portuguese, Canadian English, Canadian French, Danish, Dutch, Finnish, French, German, Greek, Italian, Norwegian Bokmal, Portuguese, Spanish, Swedish and Ukrainian. Grammar checking is available for 15 of them &mdash; there is no grammar checking for Finnish and Norwegian.
+### Language support
+
+The most popular languages used with WProofreader include: American English, British English, Canadian English, Canadian French, French, German, Italian, Greek, Spanish, Finnish, Danish, Dutch, Portuguese, Swedish, Ukrainian, Norwegian Bokmål, Brazilian Portuguese. There are, however, over 160 languages altogether, available for download from the WebSpellChecker site. Grammar checking is available for over 20 languages.
```
Comment (AnnaTomanek) [inline docs/features/spellcheck/README.md:98]:
```suggestion
The most popular languages offered by WebSpellChecker solutions include: American English, British English, Canadian English, Canadian French, French, German, Italian, Greek, Spanish, Finnish, Danish, Dutch, Portuguese, Swedish, Ukrainian, Norwegian Bokmål, Brazilian Portuguese. There are, however, over 160 languages altogether, available for download from the WebSpellChecker site. Grammar checking is available for over 20 languages.
```

**Pull request merged**